### PR TITLE
Fix multigpu training

### PR DESCRIPTION
**Pull request by Mehrad0711**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ scikit-learn = "*"
 
 [packages]
 numpy = "*"
-torch = "~=1.4"
+torch = "==1.4"
```
Comment (gcampax) [inline Pipfile]: Why this change?

File: genienlp/train.py [hunk 2]
```diff
@@ -162,7 +162,8 @@ def train_step(model, batch, iteration, opt, devices, lr_scheduler=None, grad_cl
                                                iteration > train_question_embeddings_after)
     if (iteration) % gradient_accumulation_steps == 0:
         opt.zero_grad()
-    loss, predictions = model(batch, iteration, pretraining=pretraining)
+    train_kwargs = {'iteration': iteration, 'pretraining': pretraining}
+    loss, predictions = model(batch, **train_kwargs)
```
Comment (gcampax) [inline genienlp/train.py]: It would be more readable without the ** and the dictionary.

File: Pipfile (hunk 1 above)
Comment (Mehrad0711) [reply Pipfile]: Dataparallel is buggy in torch 1.5 (https://github.com/huggingface/transformers/issues/3936)
The only known fix so far is reverting back to 1.4

**Review by gcampax — APPROVED** [review]
:+1:
